persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -115,6 +123,9 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor wit
       log.debug(s"Parsing completed for $submissionId")
       publishEvent(ParsingCompleted(submissionId))
 
+    case CompleteParsingWithErrors =>
+      log.debug(s"Parsing completed for $submissionId, errors found")
```

Comment:
Sorry, probably used the wrong variable name. What I meant is that under `case CompleteParsingWithErrors` I see the log line, but no event is fired to the event stream. It looks like it's missing something like `publishEvent(ParsingCompletedWithErrors(submissionId))`. Is this being done elsewhere? (I couldn't find it on first look)